cmd/opentelemetry/app/exporter/elasticsearchexporter/esclient/es7client.go

```diff
@@ -26,6 +26,11 @@ import (
 	"github.com/jaegertracing/jaeger/pkg/es/config"
 )
 
+const (
+	bulkES7MetaFormat = `{ "index" : { "_index": "%s" } }
```

Comment:
nit: Would be neater to just use `\n` at the end of the format string? If you did it this way because of my previous comment, I was only meaning to move the `\n` from an arg to the format string.

Reply:
As I mentioned before the `\n` cannot be used when string is defined as ``foo``. It will not be interpreted in go. The other options is to define string as `"foo"` which requires escaping all double-quotes. I find the current format easier to read.

Reply:
Ah ok, sorry missed that.

Reply:
looks hacky but I can give it a try